charts/renku/values.yaml

```diff
@@ -432,6 +432,13 @@ gitlab:
   ## Set the JupyterHub URL
   # jupyterhubUrl: "http://jupyterhub.renku.build"
 
+  ## Configuration for Renku templates
+  templates:
+    # custom: true
```

Comment:
I guess these defaults are already set in the UI `values.yaml`?

Reply:
Yes they are. Shouldn't I add `templates` anyway here?

Reply:
It's fine the way it is. I'd personally be in favour of not repeating all configuration options from the chart dependencies, but it's consistent with what we've been doing so far.

Reply:
I was going to ask the same thing - do you need `templates` to be uncommented?

Reply:
oops nvm I missed that you changed it already 😄